creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -409,17 +409,47 @@ spec:
   - name: red
     multus:
       networkName: red
------
+----
 
 At this time, `bridge` mode doesn’t support additional configuration
 fields.
 
 ______________________________________________________________________________
-*Note:* due to IPv4 address delagation, in `bridge` mode the pod doesn’t have
+*Note:* due to IPv4 address delegation, in `bridge` mode the pod doesn’t have
 an IP address configured, which may introduce issues with third-party solutions
 that may rely on it. For example, Istio may not work in this mode.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* admin can forbid using `bridge` interface type for `pod networks` via a 
```

Comment:
it's probably not a good idea to "quote" pod networks since it's not a literal from any spec or API but just an entity.